assets/model_monitoring/components/data_drift/data_drift_signal_monitor/spec.yaml

```diff
@@ -4,7 +4,7 @@ type: pipeline
 name: data_drift_signal_monitor
 display_name: Data Drift - Signal Monitor
 description: Computes the data drift between a baseline and production data assets.
-version: 0.3.15
+version: 0.3.17
```

Comment:
should be 16 or 17?

Reply:
https://github.com/Azure/azureml-assets/blob/main/assets/model_monitoring/components/data_drift/data_drift_signal_monitor/spec.yaml

17 the current version is 0.3.16, I think github isn't great for these PR diffs